core/src/dsnp/encryption.rs

```diff
@@ -47,9 +58,9 @@ mod tests {
 			83, 98, 0, 10, 234, 88, 23, 54, 23, 23, 109, 198, 111, 70, 2, 89,
 		];
 
-		let secret_key = SecretKey::generate(&mut OsRng);
-		let encrypted = SealBox::encrypt(&plain_data, &secret_key.public_key()).unwrap();
-		let decrypted = SealBox::decrypt(&encrypted, &secret_key).unwrap();
+		let key_pair = StackKeyPair::gen();
+		let encrypted = SealBox::encrypt(&plain_data, &key_pair.public_key).unwrap();
```

Comment:
Hmm, for unit test shouldn't we call `EncryptionBehavior::encrypt/decrypt` rather than `SealBox` directly?

Reply:
Would you please add more details for this. We needed a concrete implementation of EncryptionBehavior to like SealBox to use the implemented traits functions encrypt. We can call EncryptionBehavior::encrypt but from my understanding we would need to pass in a Sealbox for it to know the implementation.

Reply:
nvm, I misread the code.